_includes/themes/submarine/_navigation.html

```diff
@@ -24,34 +25,27 @@
           </ul>
         </li>
 
-
         <!-- Download -->
         <li><a href="/download.html">Download</a></li>
 
         <!-- Docs -->
-        <!--li class="docs">
-          <a href="#" data-toggle="dropdown" class="dropdown-toggle">Docs<b class="caret"></b></a>
-          <ul class="dropdown-menu">
-            <li class="title"><span><b>Release</b><span></li>
-            <li><a href="https://hadoop.apache.org/submarine/docs/0.2.0/Index/">0.2.0</a></li>
-            <li><a href="http://hadoop.apache.org/submarine/docs/">0.1.0</a></li>
-            <li role="separator" class="divider"></li>
-              <li class="title"><span><b>Snapshot&nbsp;(development)</b><span></li>
-            <li><a href="/docs/0.3.0-SNAPSHOT">0.3.0-SNAPSHOT</a></li>
-          </ul>
-        </li-->
-
```

Comment:
So we removed 0.3.0 download link?

Reply:
This snippet was commented out and it's not the download link, so we can delete it.